src/vaadin-progress-bar.ts

```diff
@@ -2,6 +2,17 @@ import { customElement } from 'lit-element';
 import { ProgressBarBase } from './vaadin-progress-bar-base';
 import { ProgressMixin } from './vaadin-progress-mixin';
 
+/**
+ * `<vaadin-progress-bar>` is a Web Component for progress bars.
+ * Content can be any HTML, including nested `<vaadin-progress-bar`.
```

Comment:
Please remove this line. It is a copy paste from `vaadin-spit-layout`, and the statement isn't true for `vaadin-progress-bar` (we don't expect it to have any content).

Reply:
done